### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivan-shpynda**
[DEMO LINK](https://ivan-shpynda.github.io/react_decompose/)

### CODE REVIEW

**Review by Grygoriy-Shytikov — APPROVED** [review]
Вигладає добре 👍
Можливо ще навігацію варто перенести до components

File: src/components/header/header.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
```
Comment (VinnieJ-2k20) [inline src/components/header/header.jsx:12]: Let's move `Navigation` and its styles to separate component

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Almost done.

Let's create component for navigation.
Also, file names for components, styles and folders usually start with capital letter.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**